examples/production/example-your-values.yaml

```diff
@@ -43,3 +43,6 @@ spiffe-oidc-discovery-provider:
         hosts:
           - oidc-discovery.production.other
 
+tornjak-frontend:
+  apiServerURL: https://spire-tornjak-backend.production.other/
```

Comment:
if we disable the ingress, how is this accessible?

Reply:
Users can enable the ingresses they want.

--set tornjak-frontend.ingress.enabled=true.